crates/routes/src/utils/scheduled_tasks.rs

```diff
@@ -557,6 +566,33 @@ async fn build_update_instance_form(
   Some(instance_form)
 }
 
+/// Updates all multi-communities from remote users
+async fn update_multi_communities(context: &Data<LemmyContext>) -> LemmyResult<()> {
```

Comment:
I don't really understand why this is necessary. 

Shouldn't they just get initially fetched on demand just like `Community`, and updates pushed after subscribing?